src/python/test/test_dxclient.py

```diff
@@ -2353,8 +2353,11 @@ def test_describe_analysis_verbose(self):
         analysis_desc_json = run("dx describe {} --json".format(dxanalysis.get_id()))
         analysis_desc_verbose_json = run("dx describe {} --verbose --json".format(dxanalysis.get_id()))
         self.assertTrue(all(key in analysis_desc_verbose for key in ["Run Sys Reqs", "Run Sys Reqs by Exec", "Merged Sys Reqs By Exec", "Run Stage Sys Reqs"]))
+        self.assertTrue(all(key in analysis_desc_verbose for key in ["ID", "Job name", "Executable name", "Class", "Workspace", "Project context"]))
```

Comment:
Here I added several keys that should be among the `defaultFields` of analysis describe